{{cookiecutter.project_name}}/.pre-commit-config.yaml

```diff
@@ -1,23 +1,82 @@
+fail_fast: false
+default_language_version:
+  python: python3
+default_stages:
+  - commit
+  - push
+minimum_pre_commit_version: 2.9.3
 repos:
--   repo: https://github.com/psf/black
-    rev: 21.12b0
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
     hooks:
-    -   id: black
--   repo: https://github.com/PyCQA/flake8
-    rev: 4.0.1
+      - id: black
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.5.1
+    hooks:
+      - id: prettier
+  - repo: https://github.com/asottile/blacken-docs
+    rev: v1.12.1
     hooks:
-    -   id: flake8
--   repo: https://github.com/pre-commit/mirrors-autopep8
+      - id: blacken-docs
+  - repo: https://github.com/pre-commit/mirrors-autopep8
     rev: v1.6.0
     hooks:
-    -   id: autopep8
+      - id: autopep8
         args: [-i]
--   repo: https://github.com/PyCQA/isort
+  - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
-    -   id: isort
--   repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
-    rev: 16762a19ddaa3a966ca7c4edd9ade6189f0df627
+      - id: isort
+  - repo: https://github.com/asottile/yesqa
+    rev: v1.3.0
+    hooks:
+      - id: yesqa
+        additional_dependencies:
+          [
+            flake8-tidy-imports,
+            flake8-docstrings,
+            flake8-rst-docstrings,
+            flake8-comprehensions,
+            flake8-bugbear,
+            flake8-blind-except,
```

Comment:
Can these go in the .flake8 file?

Reply:
If we add them there, do these have to be dev dependencies? Probably don't want that.

Reply:
they can't go in `.flake8` apparently, only local plugin.